core/embed/extmod/modtrezorui/modtrezorui-display.h

```diff
@@ -62,6 +62,17 @@ STATIC mp_obj_t mod_trezorui_Display_clear(mp_obj_t self) {
 STATIC MP_DEFINE_CONST_FUN_OBJ_1(mod_trezorui_Display_clear_obj,
                                  mod_trezorui_Display_clear);
 
+/// def sync(self) -> None:
```

Comment:
Let's not expose it to Python at all and instead call it from Rust. `LayoutObj::paint_if_requested` seems like the right place.

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/aaf6e751fc1de967251054c09e8dfbdb9d379eff